lib/serverless-tools/notifier/message.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+require "octokit"
+
+module ServerlessTools
+  module Notifier
+    class Message
+      def initialize(git_client: Octokit::Client.new(access_token: ENV["GITHUB_TOKEN"]), repo_name:, run_id:)
+        @git_client = git_client
+        @repo_name = repo_name
+        @run_id = run_id
+      end
+
+      def text_for_status(status)
+        case status
+        when "start"
+          ":building_construction: *DEPLOYING* #{deployment_details}"
```

Comment:
I think the support for short codes for emojis have different support depending on the platform. Is it worth thinking that this message could be posted anywhere? Or are we happy to say it's a message specifically for Slack? - I think if you changed these to regular emojis it should still work.

Reply:
Changed to regular emojis ✅